tests/AddCommandTest.php

```diff
@@ -76,10 +76,7 @@ public function it_detects_existing_correct_hooks()
 
         self::createHooks();
         $this->commandTester->execute([], ['verbosity' => OutputInterface::VERBOSITY_VERBOSE]);
-
-        foreach (array_keys(self::$hooks) as $hook) {
-            $this->assertContains("Hook {$hook} already ok", $this->commandTester->getDisplay());
```

Comment:
I think we still need this, but it should say `[{$hook}] is up to date` like the debug log says